vpr/src/place/RL_agent_util.cpp

```diff
@@ -36,12 +39,17 @@ void create_move_generators(std::unique_ptr<MoveGenerator>& move_generator, std:
 
         if (placer_opts.place_agent_algorithm == E_GREEDY) {
             VTR_LOG("Using simple RL 'Epsilon Greedy agent' for choosing move types\n");
-            std::unique_ptr<EpsilonGreedyAgent> karmed_bandit_agent1, karmed_bandit_agent2;
+            std::unique_ptr<EpsilonGreedyAgent> karmed_bandit_agent1, karmed_bandit_agent2, karmed_bandit_agent3;
             if (placer_opts.place_algorithm.is_timing_driven()) {
                 //agent's 1st state
                 karmed_bandit_agent1 = std::make_unique<EpsilonGreedyAgent>(NUM_PL_1ST_STATE_MOVE_TYPES, placer_opts.place_agent_epsilon);
                 karmed_bandit_agent1->set_step(placer_opts.place_agent_gamma, move_lim);
+
+                karmed_bandit_agent3 = std::make_unique<EpsilonGreedyAgent>(NUM_PL_1ST_STATE_MOVE_TYPES, placer_opts.place_agent_epsilon);
+                karmed_bandit_agent3->set_step(placer_opts.place_agent_gamma, move_lim);
+
                 move_generator = std::make_unique<SimpleRLMoveGenerator>(karmed_bandit_agent1);
+                manual_move_generator = std::make_unique<ManualMoveGenerator>(karmed_bandit_agent3);
```

Comment:
karmed_banit_agent3 and manual_+move_generator are created even if the graphics are disabled or display is off